### DISCUSSION TITLE
Custom Colors

### DISCUSSION THREAD

**Discussion by ephraimduncan** [mirror-xyz/degen#84] [Q&A]
Is it possible to use Custom Colors with the library apart from the default colors provided?

**Comment by tmm** [comment]
There's not plug-and-play way to do this right now. You could try importing `vars` and overriding the color values with [`vanilla-extract`](https://github.com/seek-oss/vanilla-extract), using your own global theme with higher specificity.

**Comment by ephraimduncan** [comment]
Could you give a simple example on how to do this? I tried with vanilla-extract. It wasn't working for me.
